src/Dom/Document.php

```diff
@@ -21,6 +26,9 @@
 use DOMNodeList;
```

Comment:
I can't reach it in the PR as a suggestion here, but the import for `DOMAttr` is not being used and can be removed.

Reply:
Removed the `use DOMAttr`.